src/components/FileUploader/index.js

```diff
@@ -181,14 +176,9 @@ FileUploader.propTypes = {
 
 FileUploader.defaultProps = {
   translate: ({ defaultText }) => defaultText,
-  hasError: false,
```

Comment:
Why did you remove all those props?

Reply:
`false` and `undefined` are both falsy, so they don't provide any value by being false, just extra code IMO. I checked that it doesn't change functionality

Reply:
👌